test/testapp.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage:"
+    echo "./testapp.sh BOX_OR_RPI_IPADDRESS bundle.tar.gz"
+    exit
+fi
+
+BOXIP=$1
+PKG=$2
+
+echo "--> Copying bundle to box on lighttpd server location"
+scp $PKG root@$BOXIP:/opt/www/
+
+echo "--> Killing app"
+curl -X POST http://$BOXIP:9998/jsonrpc -d '{"jsonrpc":"2.0","id":1,"method":"org.rdk.RDKShell.1.kill", "params":{ "client": "testclient" } }'
+echo
+
+echo "--> Remove app"
+curl -X POST http://$BOXIP:9998/jsonrpc -d '{"jsonrpc":"2.0","id":1,"method":"Packager.1.remove", "params":{ "pkgId": "testpkg" } }'
+echo
+
```

Comment:
Suggest adding a short sleep after calling uninstall as uninstall is async and can take a moment to actually uninstall the app (especially for larger apps)

Reply:
Added polling for removal status, just like when checking for installed status.